src/openrct2/drawing/Text.cpp

```diff
@@ -155,6 +155,10 @@ void gfx_draw_string_left(rct_drawpixelinfo* dpi, rct_string_id format, void* ar
 {
     DrawTextCompat(dpi, x, y, format, args, colour, TextAlignment::LEFT);
```

Comment:
```suggestion
    gfx_draw_string_left(dpi, format, args, colour, { x, y });
```
Make the old function call the new one, so we don't have duplicate code doing the same thing and also test all of them at once. When we finish migrating to the new format, we can just delete this one and there shouldn't be any side effect.

Reply:
ok thanks , i never work with clang format so it's a little bit new , for the commit i didn't understand clearly how they work from a fork sometime so i didn't really know if they are going to my fork or to this .

Reply:
It's just a script that applies some of our formatting rules. It checks out your fork and branch and applies the script. If there's a diff, it means it's not properly formatted, otherwise it would be blank.

Reply:
Can i found a version to test it in my computer before push it .

Reply:
The script we run itself is over at `scripts/check-code-formatting`. You'll need to have `clang-format` installed/on your path, that varies on how to do it depending on the OS, but a few google searches will do the trick.

`clang-format` comes bundled with `LLVM`

Reply:
@tupaschoal I'm not sure I understand. The old function with separate x and y parameters should be removed outright.

Reply:
@Gymnasiast it's easier to introduce an overload to make the migration iteratively, otherwise we have to update all calls at once. If those are a few (and already updated here), then I agree with you, but by the time I checked I think they weren't fully migrated.

In any case, if the old function is always made to call the new one, then cleaning it up later should we ever forget to would be easy.

Reply:
Considering they commented it out, I'll have to assume that they migrated all calls anyway.